src/scvi/model/_multivi.py

```diff
@@ -925,130 +960,6 @@ def differential_expression(
 
         return result
 
-    @torch.no_grad()
-    def get_protein_foreground_probability(
```

Comment:
Why removed?

Reply:
didnt see it used anywhere, but got it back now.